docs/workshop/part-ii/06-compound-promise.mdx

```diff
@@ -374,6 +374,25 @@ kubectl --context $PLATFORM apply --filename promise.yaml
 
 Great! Your App-as-a-Service Promise is now a Compound Promise that can request a PostgreSQL service. You can now test it by making a request for an App-as-a-Service with a PostgreSQL service.
 
+
+:::warning
+
+Before you proceed, ensure you have registered the platform cluster as a destination and labelled it appropriately. You can use this command to verify your destinations:
+
+```bash
+kubectl get destinations -o custom-columns=NAME:.metadata.name,LABELS:.metadata.labels
```

Comment:
any reason to avoid using `--show-labels` in this command?

Reply:
None other than lack of knowledge on my part!